.github/workflows/development-environment.yml

```
@@ -64,9 +64,9 @@ jobs:
           path: |
             ${{ steps.pip.outputs.pip-cache-dir }}
             ~/.pyenv
-          key: ${{ runner.os }}-pip-py${{ steps.python-version.outputs.python-version }}-${{ hashFiles('**/requirements-*.txt') }}
+          key: ${{ matrix.os }}-pip-py${{ steps.python-version.outputs.python-version }}-${{ steps.pip.outputs.pip-version }}-${{ hashFiles('**/requirements-*.txt') }}
           restore-keys: |
-            ${{ runner.os }}-pip-py${{ steps.python-version.outputs.python-version }}
+            ${{ matrix.os }}-pip-py${{ steps.python-version.outputs.python-version }}-${{ steps.pip.outputs.pip-version }}-
```

Typo, but don't apply this suggestion - there's multiple occurences and I mention elsewhere that you should rename the key.

```suggestion
            ${{ matrix.os }}-pip-py${{ steps.python-version.outputs.python-version }}-${{ steps.pip.outputs.pip-version }}
```

---

This is actually not a typo. This helps keep the cache of Catalina and Big Sur separate, otherwise, we would reuse the cache of Catalina for Big Sur and we would not be able to catch the regression we're intending to catch.

---

Isn't that taken case of by matrix.os though?

But anyway, looks like you fixed it?

---

Nevermind me! I misunderstood your comment.